build/build.js

```diff
@@ -64,56 +67,68 @@ const buildAllPlugin = function () {
     {name: 'gitalk', input: 'gitalk.js'}
   ]
 
-  plugins.forEach(item => {
-    build({
+  const promises = plugins.map(item => {
+    return build({
```

Comment:
should it be `push` ?

Reply:
This one's a `.map`, so it returns the whole array.

Reply:
I meant this can be with like `forEach + push` right !

this looks good too 👍